src/cycleclock.h

```diff
@@ -40,7 +40,10 @@
 extern "C" uint64_t __rdtsc();
 #pragma intrinsic(__rdtsc)
 #endif
+
+#if !defined(OS_WINDOWS)
```

Comment:
#ifndef OS_WINDOWS is consistent with the rest (though i do prefer the long form).